src/xkbcomp/rules.c

```diff
@@ -131,6 +134,10 @@ lex(struct scanner *s, union lvalue *val)
         return TOK_GROUP_NAME;
     }
 
+    /* Include statement. */
+    if (lit(s, "include"))
```

Comment:
In the commit message you wrote "The file path must be without quotes and may not start with the
literal "include"." - is it because we now treat it a a token?

Reply:
that and - I freely admit - laziness. We could allow for quotes, but then we'd have to worry about possible escaping, etc. Just easier to treat anything after the literal `include` as-is.